library/src/main/java/me/toptas/fancyshowcase/Calculator.java

```diff
@@ -81,6 +85,33 @@ public void setCirclePosition(int positionX, int positionY, int radius) {
         mHasFocus = true;
     }
 
+    public void calcAutoTextPosition(final View view, final int padding) {
+        view.post(new Runnable() {
+            @Override
+            public void run() {
+
+                // calc target view position
+                float top = roundRectTop(0, 0);
```

Comment:
You should add `mFocusCircleRadiusFactor` to the calculation. This is the coefficient value for circle radius. 
I tried this:
```
new FancyShowCaseView.Builder(this)
                .focusOn(view)
                .focusCircleRadiusFactor(4)
                .title("Focus on View with larger circle")
                .build()
                .show();

```
And here is the result:

![device-2017-05-28-110655](https://cloud.githubusercontent.com/assets/1595227/26527137/1b6fd26c-4396-11e7-8eb7-7cc4c2f24faf.png)